addons/hr/models/hr_department.py

```diff
@@ -25,6 +25,7 @@ class Department(models.Model):
     color = fields.Integer('Color Index')
 
     def name_get(self):
+        self = self.sudo()
```

Comment:
- Add a quick inline comment as to why this is needed?
- Ideally try to name the variable so we know it holds sudo rights

eg
```py
sudoed = self.sudo()
self_sudo = self.sudo()
sudo = self.sudo()
```
```